admin/server-timing.php

```diff
@@ -43,6 +43,47 @@ function perflab_add_server_timing_page() {
  * @since n.e.x.t
  */
 function perflab_load_server_timing_page() {
+	add_settings_section(
+		'output-buffering',
+		__( 'Output Buffering', 'performance-lab' ),
+		'perflab_render_server_timing_page_output_buffer_checkbox',
+		PERFLAB_SERVER_TIMING_SCREEN
+	);
+
+	add_action(
+		'admin_print_styles',
+		static function () {
+			?>
+			<style>
+				.wrap p {
+					max-width: 800px;
+				}
+
+				/* The following styles are copied from core where they normally include `.form-table td` in the selectors. */
+				.output-buffering-field {
+					margin-bottom: 9px;
+					padding-bottom: 15px;
+					line-height: 1.3;
+					vertical-align: middle;
+					font-size: 14px;
+				}
+				.output-buffering-field label,
+				.output-buffering-field p {
+					line-height: 1.4;
+				}
+				.output-buffering-field label {
+					margin: 0.35em 0 0.5em !important;
+					display: inline-block;
+					vertical-align: middle;
+				}
+				.output-buffering-field p {
+					font-size: 14px;
+				}
```

Comment:
CSS not applied to provided selector. It was override by below CSS so the text use 13px font size. If we have to use 14px font then we have to update selector `.output-buffering-field p.description`

```
p.help, p.description, span.description, .form-wrap p {
    font-size: 13px;
}
```

Reply:
Good catch! Not sure why we would want to set it to `14px` actually, why not use the regular size core would use?

Reply:
I'm OoO today. So please make whatever change you need. But I largely copied the styles from core as I recall.

Reply:
Sorry for the noise then :) Will do!